timestep_mesh.py

```diff
@@ -0,0 +1,18 @@
+import numpy as np
+
+def generate_timesteps(simulation_length, time_step) 
```

Comment:
Don't forget that python uses indentation to mark function boundaries.

Also need to end this line with a colon (:)

Reply:
Thanks for the help!

Reply:
If you've replaced this file, with the one above, you should remove it from the branch/PR: `git rm timestep_mesh.py`